amcl/src/amcl_node.cpp

```diff
@@ -287,6 +289,9 @@ class AmclNode
     tf2::TimePoint last_laser_received_ts_;
     tf2::Duration laser_check_interval_;
     void checkLaserReceived();
+
+    rclcpp::TimeSource ts;
```

Comment:
as all the other variables have a full name I'd suggest doing the same here. 
I'm concerned that it may get confusing for users as everywhere else in the code base we use `ts` to refer to type support and that above (last_laser_received_ts_), `ts` refers to a timestamp.

Reply:
Yeah, that makes sense to me.  I'll make that change before merging.